src/sbvr-api/sbvr-utils.ts

```diff
@@ -1017,16 +1097,38 @@ const runODataRequest = (req: Express.Request, vocabulary: string) => {
 			}
 
 			const prepareRequest = async ($request: uriParser.ODataRequest) => {
+				$request.translateVersions = _.clone(versions);
```

Comment:
Shouldn't `translateVersions` be a mandatory parameter for the ODataRequest after this? This would relax the typings.

Reply:
Yup, I've made it mandatory now for any requests that have gone through this step